nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -382,6 +382,22 @@ class SortOptions(BaseModel):
     order: SortOrder = SortOrder.DESC
 
 
+class Reranker(str, Enum):
+    """Rerankers
+
+    - Multi-match booster: given a set of results from different sources, e.g.,
+      keyword and semantic search boost results appearing in both sets
+
+    - Predict reranker: after retrieval, send the results to Predict API to
+      rerank it. This method uses a reranker model, so one can expect better
+      results at expenses of more latency
+
+    """
+
+    MULTI_MATCH_BOOSTER = ("multi_match_booster",)
```

Comment:
why a tuple?

Reply:
Oh, I guess I misplaced a `,` and `ruff` added the parens. Fixing it